services/cloudaccounts/k8s/k8s.go

```diff
@@ -24,6 +24,7 @@ type CloudAccountResponse struct {
 	RuntimeProtectionEnabled bool      `json:"runtimeProtection"`
 	AdmissionControlEnabled  bool      `json:"admissionControl"`
 	ImageAssuranceEnabled    bool      `json:"vulnerabilityAssessment"`
+	FLowLogsEnabled          bool      `json:"flowLogs"`
```

Comment:
I think that the json property name is different:
"magellan" in the old API, "threatIntelligenceEnabled" in the new one